### PR TITLE
Update 'About Us' page

### PR DESCRIPTION
**Pull request by sudo-rgorai**

### CODE REVIEW

**Comment by yashrsharma44** [comment]
Hey @sudo-rgorai ! Thanks for the PR. Meanwhile there are some PEP8 issues that you might want to  fix before this PR is merged, so I would suggest you installing [`pycodestyle`](https://pypi.org/project/pycodestyle/) which helps you check the PEP8 changes using a cli

**Comment by sudo-rgorai** [comment]
Hey @yashrsharma44. Thanks for the advice. I've fixed PEP8 issues. I just need to make some more changes in designations.

**Comment by sudo-rgorai** [comment]
> Hey @yashrsharma44. Thanks for the advice. I've fixed PEP8 issues. I just need to make some more changes in designations.

Done. Please check the PR.

**Comment by mukul-mehta** [comment]
I've tested the PR locally, everything seems to be working.

We can merge once reviews are complete.
@yashrsharma44 @americast @xypnox please review

**Review by yashrsharma44 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -7,3 +7,5 @@
 
 
 Edit the header at `header.js` and footer at `footer.js` at the root.
+
+Update About Us page by running `about\script.py`.
```
Comment (americast) [inline README.md]: Shouldn't this be `about/script.py`? :sweat_smile:

Comment (sudo-rgorai) [reply README.md]: Not unless you're using windows 😅. My bad. Should I change it?

Comment (americast) [reply README.md]: Definitely. It's outright "incorrect" 😛.
 @mukul-mehta seems like you guys aren't grooming your juniors well :wink:

Comment (sudo-rgorai) [reply README.md]: @americast Apologies. I've changed it.

Comment (mukul-mehta) [reply README.md]: Sorry, I didn't have a look at this. Just tested the code. Will be sure to do that from next time :smile:

Comment (americast) [reply README.md]: @mukul-mehta code review is something else. I would want to stress on proper grooming 😛:wink:

**Review by americast — APPROVED** [review]

**Review by mukul-mehta — APPROVED** [review]

**Review by xypnox — APPROVED** [review]

**Pull request merged**